packages/applications-service-api/src/repositories/document.ni.repository.js

```diff
@@ -14,7 +14,7 @@ const fetchDocuments = async (requestQuery) => {
 	const dbQuery = {
 		where,
 		order: [['date_published', 'DESC'], ['id']],
-		offset: (requestQuery.page - 1) * requestQuery.itemsPerPage,
+		offset: (requestQuery.page - 1) * requestQuery.itemsPerPage || 0,
```

Comment:
Is this because we normally default the itemsPerPage in the FE and now it may not be sent when calling from the function?

Reply:
Yes, otherwise sequalize threw an error because we were passing undefined (on NaN - can't recall which). But this wouldn't happen from the usual endpoints as they always have defaults.